atd-vze/src/views/Crashes/crashGridTableParameters.js

```diff
@@ -203,28 +260,6 @@ export const crashGridTableAdvancedFilters = {
     icon: "map-marker",
     label: "Geography",
     filters: [
-      {
-        id: "geo_geocoded",
-        label: "Has Been Geocoded",
-        filter: {
-          where: [
-            {
-              is_manual_geocode: "_eq: true",
-            },
-          ],
-        },
-      },
-      {
-        id: "geo_confirmed_coordinates",
-        label: "No CRIS Coordinates",
-        filter: {
-          where: [
-            {
-              has_no_cris_coordinates: "_eq: true",
-            },
-          ],
-        },
-      },
```

Comment:
Xavier is ok with this change. we will revisit these columns in https://github.com/cityofaustin/atd-data-tech/issues/18542